format-common/src/main/java/io/cdap/plugin/format/plugin/AbstractFileSink.java

```diff
@@ -88,32 +88,24 @@ public void configurePipeline(PipelineConfigurer pipelineConfigurer) {
     }
 
     FileFormat format = config.getFormat();
-    ValidatingOutputFormat validatingOutputFormat =
-      pipelineConfigurer.usePlugin(ValidatingOutputFormat.PLUGIN_TYPE, format.name().toLowerCase(),
-                                   format.name().toLowerCase(), config.getRawProperties());
+    ValidatingOutputFormat validatingOutputFormat = getValidatingOutputFormat(pipelineConfigurer);
     FormatContext context = new FormatContext(collector, pipelineConfigurer.getStageConfigurer().getInputSchema());
     validateOutputFormatProvider(context, format, validatingOutputFormat);
   }
 
+  public ValidatingOutputFormat getValidatingOutputFormat(PipelineConfigurer pipelineConfigurer) {
+    FileFormat format = config.getFormat();
+    return
+        pipelineConfigurer.usePlugin(ValidatingOutputFormat.PLUGIN_TYPE, format.name().toLowerCase(),
```

Comment:
style: should have something on the previous line besides just return. Also, indentation for a new line should be 2 spaces and not 4.

Reply:
Fixed.